### PR TITLE
Fixed #38: Add ability to share a page

### LINKED ISSUE #38: Add ability to share a page
**I'm submitting a:**  
- [ ] bug report
- [x] feature request

**Description**:

Incorporate the ability to share a page/ search result to social media platforms. The sharing icons may appear below the search bar that is adjacent to the result information (`Showing 1 results for: linear search` for search term `linear search`).

Initially, you may support 3 platforms namely: Twitter, Google+ and Facebook.

This will be a convenient feature to share a result with the user's community and will be a small step towards moving searching to social which is one of our core missions.

### PR DESCRIPTION
**Pull request by sakshee-19**
**Checklist**

- [X] My branch is up-to-date with the upstream `develop` branch.
- [ ] I have added necessary documentation (if appropriate).

**Which issue does this PR fix?**: 
fixes: #38 

**Why do we need this PR?**:
The changes proposed in this PR adds ability to share search results to social networks.
screenshot from 2018-02-12 20-16-30

### CODE REVIEW

File: search/templates/cosmos/header.html [hunk 1]
```diff
@@ -8,6 +8,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
     <link href="https://fonts.googleapis.com/css?family=Roboto+Mono" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script type="text/javascript" src="//platform-api.sharethis.com/js/sharethis.js#property=5a8129cbe45fbb001342fed3&product=custom-share-buttons"></script>
```
Comment (AdiChat) [inline search/templates/cosmos/header.html]: You can move this after the footer where the other script tags are residing.

**Comment by AdiChat** [comment]
The design looks perfect. Kindly make the minor change and it will be good to be merged. 👍

File: search/templates/cosmos/header.html (hunk 1 above)
Comment (sakshee-19) [reply search/templates/cosmos/header.html]: :+1:

**Comment by Pl4gue** [comment]
Thanks a lot for the great contribution! :+1:

**Pull request merged**